src/crimson/common/errorator.h

```diff
@@ -79,7 +79,7 @@ class error_t {
 template <class ErrorT, ErrorT ErrorV>
 struct unthrowable_wrapper : error_t<unthrowable_wrapper<ErrorT, ErrorV>> {
   unthrowable_wrapper(const unthrowable_wrapper&) = delete;
-  static constexpr unthrowable_wrapper instance{};
```

Comment:
also, might want to revise the title and body of commit message to summarize the change, and explain the reason why we want to change this way.